README.md

```diff
@@ -116,6 +116,33 @@ display.Show("Hello World!")
 display.VerifyWasCalledOnce().Show("Hello World!")
 ```
 
+Tracking the pegomock tool in your project
+------------------------------------------
+
+Go modules allow to pin not only a package but also a tool (that is, an executable). The steps are:
+
+1. Use a file named `tools.go` with contents similar to this:
+```go
+// +build tools
+
+// This file will never be compiled due (see the build constraint); it is used
+// simply to record dependencies on build tools.
+// See https://github.com/go-modules-by-example/index/blob/master/010_tools/README.md
+
+package tools
+
+import (
+	_ "github.com/petergtz/pegomock/pegomock"
```

Comment:
One thing to note, this does not work with 1.13.

```
go get -tags tools ./internal/...
internal/tools.go:6:2: import "github.com/petergtz/pegomock/pegomock" is a program, not an importable package
```

Reply:
@jpopadak Could you elaborate? I just tried this and it works for me. Here's what I've done:
```shell
$ mkdir -p /home/peter/tmp/blump
$ cd /home/peter/tmp/blump
$ git init -q
$ git remote add origin https://github.com/petergtz/blump
$ go mod init
go: creating new go.mod: module github.com/petergtz/blump
$ export GOBIN=$PWD/bin
$ export PATH=$GOBIN:$PATH

$ cat tools.go
// +build tools

package tools

import (
	_ "github.com/petergtz/pegomock/pegomock"
)

$ cat go.mod
module github.com/petergtz/blump

go 1.13

require (
	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
	github.com/petergtz/pegomock v2.6.0+incompatible // indirect
	golang.org/x/tools v0.0.0-20190927191325-030b2cf1153e // indirect
	gopkg.in/alecthomas/kingpin.v2 v2.2.6 // indirect
)

$ cat main.go
package main

import "fmt"

func main() {
	fmt.Println("Hello")
}

$ go install github.com/petergtz/pegomock/pegomock
go: finding github.com/petergtz/pegomock v2.6.0+incompatible
go: downloading github.com/petergtz/pegomock v2.6.0+incompatible
go: extracting github.com/petergtz/pegomock v2.6.0+incompatible
go: finding gopkg.in/alecthomas/kingpin.v2 v2.2.6
go: finding golang.org/x/tools latest
go: downloading gopkg.in/alecthomas/kingpin.v2 v2.2.6
go: downloading golang.org/x/tools v0.0.0-20190927191325-030b2cf1153e
go: extracting gopkg.in/alecthomas/kingpin.v2 v2.2.6
go: extracting golang.org/x/tools v0.0.0-20190927191325-030b2cf1153e
go: finding github.com/alecthomas/template latest
go: finding github.com/alecthomas/units latest
go: downloading github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
go: downloading github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d
go: extracting github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
go: extracting github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d

$ which pegomock 
/home/peter/tmp/blump/bin/pegomock
```